### DISCUSSION TITLE
QDate range always model-value is object from to

### DISCUSSION THREAD

**Discussion by romkhram** [quasarframework/quasar#11452] [General - Components / Directives / etc]
if QDate range both params has same date, model-value does not object and becomes string
if {from to} has different values of same date, calendar display no active dates
can i set some atribut or another way to olways use 2 params {from to} and see active date in calendar?

**Comment by MilosPaunovic** [comment]
You could use `@range-end` event to handle the data how you want.
[https://codepen.io/milospaunovic/pen/xxLNeqK?editors=1011](https://codepen.io/milospaunovic/pen/xxLNeqK?editors=1011)

**Reply by romkhram** [reply]
> You could use `@range-end` event to handle the data how you want. https://codepen.io/milospaunovic/pen/xxLNeqK?editors=1011

thank you? but if i need reactivity? https://codepen.io/romkhram/pen/Exvzzgg?editors=1011 i cant use v-model with to/from same date
